pyproject.toml

```diff
@@ -37,20 +37,20 @@ exclude = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
-httpx = "^0.16.1"
-starlette = "^0.13.8"
-uvicorn = "^0.12.2"
+python = "^3.9"
```

Comment:
I really don't want to update the `python` version, as it locks Python to `>=3.9 <4.0`, which I don't want. The reason I kept it on `^3.6` was to be able to use versions `>=3.6 <4.0`. You can read more about dependency specification [here](https://python-poetry.org/docs/dependency-specification/)